doc/readme.md

```diff
@@ -310,7 +317,8 @@ Create a Smartcar OAuth client for your application.
 | options.clientSecret | <code>String</code> |  | The application's client secret. |
 | options.redirectUri | <code>String</code> |  | Redirect URI registered in the [application settings](https://developer.smartcar.com/apps). The given URL must exactly match one of the registered URLs. |
 | [options.scope] | <code>Array.&lt;String&gt;</code> | <code>all</code> | List of permissions your application requires. This will default to requiring all scopes. The valid permission names are found in the [API Reference](https://smartcar.com/docs#get-all-vehicles). |
-| [options.development] | <code>Boolean</code> | <code>false</code> | Launch Smartcar auth in development mode to enable the mock vehicle brand. |
+| [options.testMode] | <code>Boolean</code> | <code>false</code> | Launch the Smartcar auth flow in test mode. [API Reference](https://smartcar.com/docs#request-authorization). |
+| [options.development] | <code>Boolean</code> | <code>false</code> | DEPRECATED: Launch Smartcar auth in development mode to enable the mock vehicle brand. |
```

Comment:
mock vehicle brand no longer exists so this may be confusing. Thoughts on "...development mode to enable mock vehicle brands."?